### DISCUSSION TITLE
Adjustments for MegaXPro 2560 R3

### DISCUSSION THREAD

**Discussion by kdawg8** [sanni/cartreader#158] [Need help]
I used the link provided in the wiki to order the Arduino board and ended up getting a MegaXPro 2560 R3 which has some differences in the pcb layout to the example and I'm not entirely sure where the red wire should be soldered as I'm having a hard time following the traces on the board and can't find a layout online.  Do you know which side of the chip I marked I should put the red wire?  (or if that's even the right chip to solder it to?)  Thanks! 

![Marked up Arduino Picture](https://user-images.githubusercontent.com/83562863/119572129-86789100-bd6f-11eb-9de3-27dd90b7526c.jpg)

**Comment by sanni** [comment]
I can't see the path the traces take in your picture so I will just describe it.

The 5V wire needs to be soldered to the end of the fuse. The 5V voltage comes from the USB port, passes through the fuse and then would normally go to the de-soldered transistor(the tiny chip), instead it now goes into the wire and from there to the Cart Reader shields 5V pad.

The GND wire connects to any GND point on the Arduino, you can use the datasheet of the de-soldered voltage regulator(the big chip) to check what pad is GND or use a multimeter or if you are not sure, just leave the GND wire off, it's not needed.

**Reply by kdawg8** [reply]
Ok got it.  T05 measures a few tenths of an Ohm in resistance, the left side goes to the USB V input, and the right-hand side goes to the desoldered transistor; so I'll solder the red wire on the right-hand side of T05.  Thanks for your help!
